ui/src/dashboards/components/DashboardHeader.js

```diff
@@ -59,6 +62,16 @@ const DashboardHeader = ({
                   Rename
                 </button>
               : null}
+            {dashboard
```

Comment:
At some point I think we need to refactor this and other Dashboard components with the assumption that there will always be a Dashboard present. I don't like having to perform this check all the time